### ISSUE TITLE
WeightStandardizedConv2d will cause inf

### ISSUE THREAD

**Issue by Runist** [lucidrains/denoising-diffusion-pytorch#233]
I found 
after `F.conv2d(x, normalized_weight, self.bias, self.stride, self.padding, self.dilation, self.groups)` calculation，output will have inf.
Large Learning rate will be easier to reproduce.

**Comment by lucidrains** [comment]
I ran into issues with that with amp, so I removed it already in a recent commit

**Comment by Runist** [comment]
> 

thank for reply, I will update code

**Comment by Runist** [comment]
> I ran into issues with that with amp, so I removed it already in a recent commit
How many steps do I usually need to train to see good results? I have been training on noise maps all along

**Comment by lucidrains** [comment]
@Runist as long as possible, def more than 20k steps
